_content/docs/editing.md

```diff
@@ -39,3 +39,33 @@ Getting the site running on your machine locally is easy to do on Mac and Linux.
+
+## Years ##
+
+Years are one of the main attributes that are used within the project. These are important to get right and important for many different reasons. For more information on the correct formatting and usage, see the [Years](https://history.newtheatre.org.uk/docs/year/) page.
+
+## Committees ##
+
+New committees are usually added by an [owner](https://github.com/orgs/newtheatre/people) at the start of each academic year. Committee files are stored in the committees folder. We do get submissions from alumni with information regarding past committees and these can be added by anyone. For detailed information on how to format a committee file, see the [Committees](https://history.newtheatre.org.uk/docs/committee/) page.
+
+## Person Lists ##
+
+These are used mainly to populate the cast and crew lists for shows. They are formatted in a particular way and this must be kept to otherwise other areas of the site could break or throw up errors as person lists are used to populate the person collection. For detailed information on formatting person lists, see the [Person List](https://history.newtheatre.org.uk/docs/person-list/) page.
+
+## Link Lists ##
+
+These are used particularly for reviews placed within show files, but the list of possible links is expanding as we grow the site, so could change in the future. For formatting details see the [Link List](https://history.newtheatre.org.uk/docs/link-list/) page.
+
+## Trivia ##
+
+One of the things we love collecting for the History Project are little bits of trivia or anecdotes regarding shows. Be this a last minute cast change for one night or something that happened that amused the cast, these stories are a vital part of what makes the NNT the NNT. To find out how to format these sections, visit the [Trivia List](https://history.newtheatre.org.uk/docs/trivia-list/) page.
+
+## Photos and Assets ##
+
+The History projects visual records are stored in a [SmugMug Site](https://photos.newtheatre.org.uk/). Anyone can view the production shots from any of our shows, but the other archives are password protected. Any show videos, such as Trailers or Behind the Scenes videos are also stored here. If you have any items you feel would be useful to the History Project, please see the [Sending In Images](https://history.newtheatre.org.uk/upload/)page.
```

Comment:
Technically not password protected, just not advertised publicly. If you have the URL to an album you can view it. Only reason we did this was to clean up the home page of the `photos.new…` site.